### DISCUSSION TITLE
覺得手機在「偷聽」你說話嗎？

### DISCUSSION THREAD

**Discussion by GuanYann** [1101-GS4719-C/Python-Community-GS4719-C#1226] [General]
你有覺得被手機偷聽的經驗嗎? 在完全沒有使用手機的情況下，只是閒聊某項商品、品牌，之後滑手機卻大量收到有關這些東西的廣告。即使有媒體實測並不會有偷聽這樣的現象，若還是覺得毛毛的話，可以參考以下的動作。
1..不要下載來源不明的應用程式
2.確保手機的麥克風和其他感應器被正確配置和設置
3.定期檢查手機是否有任何安全更新
4.在需要時關閉麥克風和其他感應器的應用程式權限

https://udn.com/news/story/7098/7087585?from=udn-catebreaknews_ch2

**Comment by Hsucuchia** [comment]
在這個數位時代，個人資料隱私成為一個不容忽視的問題，因此保護個人隱私是非常重要的。雖然媒體實測顯示手機並不會進行偷聽，但我們仍然應該采取一些措施來保護自己的隱私。舉例來說，不要下載不明來源的應用程式，確保手機的設置和配置都正確，並定期檢查安全更新，關閉不必要的應用程式權限等。保護個人隱私不僅僅是一個個體的問題，也是社會的問題!

**Comment by curtis1017** [comment]
在iphone上，有一個特別的功能，那就是Siri，只要我們說 Hey Siri 時，你的Siri就會馬上出來為你服務，從這點我就可以知道其實手機有在做監聽這個動作，甚至有時候我根本沒有要Siri出來，他還是會自己跑出來。

**Reply by KozakHou** [reply]
總算找到同道中人，我一直都在懷疑有再被錄音，有時只是跟朋友聊天提到某產品(沒有上網查)，結果FB跟蝦皮都會推薦那類商品。
好像就跟之前史諾登爆的稜鏡計畫一樣呢

**Comment by KevinCheng42** [comment]
除了被「偷聽」外，我們的隱私也會因為在瀏覽器上的搜尋紀錄而被記錄下來，所以我覺得我們應該要提高警覺，確認社群媒體或瀏覽器的設定，避免洩漏個人資料。

**Comment by DuxtinTai** [comment]
我完全同意你提到的四個動作。特別是關於手機應用程式權限的問題，現在很多應用程式都需要訪問手機的麥克風、相機和位置等感應器來提供更好的服務和體驗。但是這些權限也可能被濫用，威脅到我們的隱私和安全。

因此，定期檢查手機上的應用程式權限是非常重要的。如果有一些應用程式權限不必要，或者我們不太信任某個應用程式，就應該關閉相關的權限。這樣可以保護我們的隱私和安全，減少不必要的風險。

同時，我們也應該避免下載來源不明的應用程式，以免下載到帶有病毒或惡意程式的應用程式。確保手機的麥克風和其他感應器被正確配置和設置也很重要，可以避免一些不必要的風險。

總之，保護我們的隱私和安全是非常重要的。即使我們可能會感到毛毛的，但是我們可以採取相應的措施來減少不必要的風險。

**Comment by CCello** [comment]
補充幾點來避免手機偷聽:
1.使用加密通訊應用程式：使用加密通訊應用程式，如WhatsApp、Signal和Telegram等，以確保通訊內容得到加密保護。這可以防止黑客和其他人在中途攔截你的通訊。
2.關閉藍牙和Wi-Fi：如果不使用藍牙或Wi-Fi，請務必關閉它們，以防止攻擊者通過這些通訊協議來訪問手機。
3.不要使用公共Wi-Fi：公共Wi-Fi網絡通常不安全，因為攻擊者可以使用它們來攻擊你的手機。如果必須使用公共Wi-Fi，請使用虛擬私人網絡（VPN）來加密通訊內容。

**Comment by Jack-1D** [comment]
一開始看到類似的廣告真的會覺得也太巧了吧! 後來才漸漸發現好像真的被監聽了。像是有一次我附近剛好有人在看象棋的影片，後來我打開YouTube就一直被推薦象棋的影片，明明我都不看象棋的......

**Comment by Evening218** [comment]
我之前曾經就這個問題查詢過一些資料，比較有印象的說法是我們對話的對象通常是家人或朋友之類的人，而應用程式的演算法似乎會根據和我們有關係的人的搜尋紀錄推送廣告或相關貼文。當然不排除監聽的可能，但如果已經關閉應用程式的麥克風權限還是覺得手機聽得到自己說過的話，也許就是因為我們周遭的人最近在關注這樣東西所以就連我們也被推送資訊了

**Comment by Chiuhsianglin** [comment]
之前可能跟朋友聊到某些東西之後，不管是在ig還是其他地方，都會跳出相關的廣告，讓我覺得非常的恐怖，之前我也看過啾啾鞋有出一部影片是有關於這個的測試，我覺得蠻有趣的，可以去看一下這個影片。
https://www.youtube.com/watch?v=K7jm3kTcbK0

**Comment by pigeon9975** [comment]
關於被監聽我也有過類似經驗！之前跟同學聊天聊到相機時，我的FB、IG就開始大量推薦我相機廣告，明明之前從來不會這種類型的資訊的...除此之外，瀏覽器的搜尋等也會改變推薦廣告，仔細想想真的蠻毛的

**Comment by Chienyuuu6** [comment]
我也覺得我有被監聽，每次我跟朋友分享一個想買的東西，或是查了某個特定網站，下一秒在我的社群媒體上就會出現大量的相關廣告，超級猛（Pypal聊天機器人回覆我的速度跟內容也很猛⋯）

感謝作者跟大家分享一些隱私防範ㄉ方法，其實我對於手機監聽這件事情蠻中性的，一方面是可以讓我們被動得到更多想知道的資訊，事情一體兩面～～

**Comment by ziihui** [comment]
謝謝這篇提供的資訊！馬上去設定⋯🙌🏻
前幾天我跟朋友介紹一位明星，他說他從來沒聽過他，也沒在社群軟體看過><
結果我說完後，他的ig搜索就開始出現了⋯yt也出現她的相關影片🥹
我們都嚇到了⋯
除此之外明明想買一樣東西，然後只是蝦皮搜尋，結果ig、臉書的廣告就瘋狂跳出我搜尋的東西，真的有點困擾🥹

**Comment by Tzuhsuannnnn** [comment]
很常在跟朋友聊天聊到想買甚麼東西時，手機上各個網站就會開示推送相關產品。雖然有安全方面疑慮，但反過來想，能快速獲取自己需要的資訊也蠻方便的...

**Comment by YoYooHuang** [comment]
真的！我自己也有這種感覺，才剛跟朋友聊完某雙球鞋到底該不該買，晚上滑ig的時候就跳出這雙球鞋的廣告，我看到當下也覺得毛骨悚然。希望這科技日新月異的情況下，大家能遵守科技倫理，不要違背科技發展的初衷。

**Comment by orianna901001** [comment]
雖然我一直都感覺手機在偷偷的監聽，但前陣子看到新聞在播報這個消息時，才發現3C的便利性其實會偷偷竊取我們很多機密。
PS 如果想要預防竊聽你可以關閉App的麥克風。

**Comment by Jamesmoneya** [comment]
我曾經在一天之中被廣告推撥了三種個人化的廣告，有一次再跟同學討論是否要吃牛排，我們討論牛排的吃法講出牛排的關鍵字接近100次左右，正當我們結束要各自回去訰被出發時，我拿起我的手機，剛好訊息跳出來，GOOGLE MAP上寫推薦您附近好吃的牛排店，我真的怕的

**Comment by sjh501** [comment]
在安裝應用程式時我都會特別點開權限要求，確定應用程式的要求是合理的才會安裝。另外在安裝過程中也會跳出同意權限的對話框，手機的應用程式設定裡面，每個應用程式的每個權限要求都能個別做進階的設定（內建的應用程式也可以）。閒閒沒事點進去看一看，不僅打發時間也保護到自己的隱私。

**Comment by alicia0725** [comment]
感受最深的就是跟朋友聊天的話題很容易出現在Youtube跟Fb的廣告，我原本認爲這樣很方便，不用再特別去找，但看完這篇文章，才發現手機一直在聽我的生活，會趕快去設定的！
